pustaka/variables.tex

```diff
@@ -0,0 +1,22 @@
+% Atur variabel berikut sesuai namanya
+
+\newcommand{\name}{Elon Reeve Musk}
+\newcommand{\authorname}{Musk, Elon Reeve}
+\newcommand{\nickname}{Elon}
+\newcommand{\advisor}{Nikola Tesla, S.T., M.T}
+\newcommand{\coadvisor}{Wernher von Braun, S.T., M.T.}
+\newcommand{\nrp}{0123 20 4000 0001}
+\newcommand{\advisornip}{18560710 194301 1 001}
+\newcommand{\coadvisornip}{18560710 194301 1 001}
+\newcommand{\tatitle}{KALKULASI ENERGI PADA ROKET LUAR ANGKASA BERBASIS \emph{ANTI-GRAVITASI}}
+\newcommand{\engtatitle}{\emph{ANTI-GRAVITY BASED ENERGY CALCULATION ON OUTER SPACE ROCKETS}}
+\newcommand{\place}{Surabaya}
+\newcommand{\studyprogram}{Teknik Dirgantara}
+\newcommand{\engstudyprogram}{Aerospace Engineering}
+\newcommand{\faculty}{Teknologi Dirgantara}
+\newcommand{\facultyshort}{FTMD}
+\newcommand{\engfacultyshort}{FMAE}
+\newcommand{\engfaculty}{Aerospace Technology}
+\newcommand{\department}{Teknik Dirgantara}
+\newcommand{\engdepartment}{Aerospace Engineering}
+\newcommand{\coursecode}{TD123456}
```

Comment:
could you ordered it to be more organized? e.g. put `engVar` right after `var`, separate each group by an empty line, etc.

Reply:
okey